### PR TITLE
Profile screen (EXPOSUREAPP-6029)

### PR DESCRIPTION
**Pull request by mtwalli**
- Creates Profile Qr Code screen
- Simple V-Card format string replacement

### TODO
- [x] generate Qr code from v-card format 
- [x] Add some unit tests

### CODE REVIEW

**Review by janetback — DISMISSED** [review]
UA reviewed

File: Corona-Warn-App/src/main/res/layout/rat_profile_qr_code_fragment.xml [hunk 1]
```diff
@@ -1,26 +1,197 @@
...
+                    android:layout_marginHorizontal="24dp"
+                    android:layout_marginTop="8dp"
+                    android:orientation="vertical"
+                    android:padding="16dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/info_box"
+                    app:layout_constraintVertical_chainStyle="packed">
+
+                    <TextView
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/rat_profile_qr_code_fragment.xml:169]: Spacing between the lines should be little bigger according to Figma

**Review by AlexanderAlferov — DISMISSED** [review]
LGTM 🚀

**Review by axelherbstreith — APPROVED** [review]
Looks really nice! Great work 👍

**Review by AlexanderAlferov — APPROVED** [review]

**Review by janetback — COMMENTED** [review]
UA reviewed

**Review by janetback — COMMENTED** [review]
UA reviewed

**Pull request merged**